BAZEL.md

```diff
@@ -1033,10 +1033,9 @@ All direct Scala and Java dependencies are listed explicitly in the file
 `maven_install` repository rule calls Coursier to perform transitive dependency
 resolution and import the required artifacts into the Bazel build.
 
-The resolved versions are pinned in the file `maven_install.json`. Execute
+The resolved versions are pinned in the file `maven_install_2.13.json`. Execute
```

Comment:
Are we keeping this because we expect to upgrade to Scala 3.x at some point?

Reply:
I hope so yeah. And it doesn’t really seem to cause any harm to have the suffix.